### PR TITLE
fixing (Missing Nullable Statement in MudBlazor results in Build warning #105)

### PR DESCRIPTION
**Pull request by joe97z**
adding if statement for the value if is not "Null"

### CODE REVIEW

**Comment by Apollo3zehn** [comment]
Hi, thanks for your PR. The problem with your proposed solution is that now null values are not being set anymore although they are valid values in the general case. 

But at the same time the property `MudAutocomplete<T>.ResetValueOnEmptyText` property is left at the default value (false) which means that there will never be a `null` value.

In my opinion the MudBlazor project did not properly null-annotate their `SetValueAsync` method to allow for null values and so we can only wait for an update from them.

So to summarize:
- On the one hand, your solution is OK because there will never be null values
- And on the other hand it is a bit misleading because null values are OK in the general case and ideally MudBlazor will change the signature of the `SetValueAsync` method

I prefer reevaluating issue #105 as soon as https://github.com/MudBlazor/MudBlazor/issues/6535 is solved. Thanks again for your efforts!

**Comment by dennisrahmen** [comment]
@joe97z @Apollo3zehn just FYI you might want to check again when v8 is released.

This https://github.com/MudBlazor/MudBlazor/pull/10127 was just merged as part of the general effort for nullable annotation.
